sidebars.js

```diff
@@ -90,8 +90,8 @@ const sidebars = {
         "tutorials/first-coin",
         "tutorials/first-fungible-asset",
         "tutorials/first-move-module",
-        "tutorials/first-dapp",
         "tutorials/first-multisig",
+        "tutorials/build-e2e-dapp/index",
```

Comment:
Nice, we may want a redirect for the purposes of SEO, and old links.  But, I'm glad we're getting rid of the old dapp example.